README.md

```diff
@@ -26,6 +28,7 @@ _gpgmailer_ depends on one other piece of the Parkbench project, which must be i
 
 ## Steps to Build and Install
 
+0.   Install and configure your local MTA.
```

Comment:
Please reorder all number to start with one. (Richard Stallman would disagree.) I would also link to the postfix.md here.